File: Resources/Script Group/Base64 Decode.py
from java.util import Base64
from java.nio.charset import StandardCharsets
from java.lang import String

def invoke(msg):
    base64EncodedData = msg.get("data.base64")
    if base64EncodedData is not None:
        dataDecoded = Base64.getDecoder().decode(base64EncodedData)
        dataString = String(dataDecoded, StandardCharsets.UTF_8)
        msg.put("data.decoded", dataString)
    return True

File: Resources/Script Group/Base64 Decode.js
importPackage(Packages.java.util);
importPackage(Packages.java.nio.charset);

function invoke(msg)
{
    var data = msg.get("data.base64");
    var base64DecodedData = Base64.getDecoder().decode(data);
    var dataString = new java.lang.String(base64DecodedData, StandardCharsets.UTF_8);
    msg.put("data.decoded", dataString);
    return true;
}